### DISCUSSION TITLE
Elastic Agent alerts

### DISCUSSION THREAD

**Discussion by 8bitjoe** [Security-Onion-Solutions/securityonion#10556] [2.4] [answered]
Greetings, 

What alerting mechanism will be used now with elastic agent? 
With Wazuh it alerted on the SOC UI for windows events and so on on top of the ossec alerts... with the new agent how will the HIDS alerting on SOC work?
does it need the elastic defend integration?  playbook rules? Osquery Packs?
The Documentation for this is sparse to non-existent, i was hoping you guys could shed some light on this.

**Comment by 8bitjoe** [comment]
Any thoughts on this?

**Comment by defensivedepth** [comment, marked as answer]
Playbook (running Elastalert in the background) will be the primary alerting engine for logs ingested via Elastic Agent. We will be adding more documentation around this as we get closer to GA.

**Reply by 8bitjoe** [reply]
thank you, i know its silly to ask for an ETA on GA but do you guys believe that it will be available this year?

**Reply by dougburks** [reply]
> i know its silly to ask for an ETA on GA but do you guys believe that it will be available this year?

Yes.

**Comment by asmaaelkandoussi** [comment]
Hi , 
any update how to configure alert

**Reply by asmaaelkandoussi** [reply]
Any update how to configure alert 
same problem with SO 2.4.10

**Reply by dougburks** [reply]
@asmaaelkandoussi From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
_Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion._

I've responded to your new discussion at https://github.com/Security-Onion-Solutions/securityonion/discussions/11446.
